src/elfpy/utils/time.py

```diff
@@ -39,6 +39,12 @@ def norm_days(days, normalizing_constant=365):
     return days / normalizing_constant
 
 
+def calc_time_stretch(apy):
```

Comment:
I think this should probably live with the pricing model. Right now it's the same, but we've talked a lot about potentially changing this up (e.g. making it dynamic) or removing this for HyperDrive.